cypress/fixtures/veggie_request.json

```diff
@@ -0,0 +1,26 @@
+{
+	"category": {
+		"title": "veggie",
+		"products": [
+			{
+				"id": 1,
+				"name": "Hawaii",
+				"description": "ham, tomato sauce, pineapple and cheese",
+				"price": "179kr"
+			},
+
+			{
+				"id": 2,
+				"name": "Margherita",
+				"description": "tomato sauce, cheese",
+				"price": "129kr"
+			},
+			{
+				"id": 3,
+				"name": "Vesuvio",
+				"description": "tomato sauce, cheese and ham",
+				"price": "139kr"
+			}
+		]
+	}
+}
```

Comment:
Format this file and fix the indentation

Reply:
This is formatted correctly in our VSC.... dont know why its not that in Github?

![Screenshot from 2020-11-27 15-01-02](https://user-images.githubusercontent.com/72013716/100456930-679b8980-30c1-11eb-8872-a9e5680afbdf.png)